.github/workflows/mac_build_test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           cd ${HOME}
           mkdir -pv moab/bld
           cd moab
-          git clone https://bitbucket.org/fathomteam/moab -b 5.3.0 --depth 1 --shallow-submodules
+          git clone https://bitbucket.org/fathomteam/moab -b 5.4.1 --depth 1 --shallow-submodules
```

Comment:
I'd rather not move the MOAB version ahead for MacOS if we aren't moving it forward for the other platforms.

Reply:
Oh.. I have missed it.